Mathlib/Analysis/LocallyConvex/AbsConvex.lean

```diff
@@ -40,6 +42,95 @@ open NNReal Pointwise Topology
 
 variable {𝕜 E F G ι : Type*}
 
+section AbsolutelyConvex
+
+variable (𝕜) [SeminormedRing 𝕜] [SMul 𝕜 E] [SMul ℝ E] [AddCommMonoid E]
+/-- A set is absolutely convex if it is balanced and convex. -/
+def AbsConvex (s : Set E) : Prop := Balanced 𝕜 s ∧ Convex ℝ s
+
+variable {𝕜}
+
+theorem AbsConvex.empty : AbsConvex 𝕜 (∅ : Set E) := ⟨balanced_empty, convex_empty⟩
+
+theorem AbsConvex.univ : AbsConvex 𝕜 (Set.univ : Set E) := ⟨balanced_univ, convex_univ⟩
+
+theorem AbsConvex.inter {s : Set E} {t : Set E} (hs : AbsConvex 𝕜 s) (ht : AbsConvex 𝕜 t) :
+    AbsConvex 𝕜 (s ∩ t) := ⟨Balanced.inter hs.1 ht.1, Convex.inter hs.2 ht.2⟩
+
+theorem absConvex_sInter {S : Set (Set E)} (h : ∀ s ∈ S, AbsConvex 𝕜 s) : AbsConvex 𝕜 (⋂₀ S) :=
+  ⟨balanced_sInter (fun s hs => (h s hs).1), convex_sInter (fun s hs => (h s hs).2)⟩
+
+variable (𝕜)
+
+/-- The absolute convex hull of a set `s` is the minimal absolute convex set that includes `s`. -/
+@[simps! isClosed]
+def absConvexHull : ClosureOperator (Set E) :=
+  .ofCompletePred (AbsConvex 𝕜) fun _ ↦ absConvex_sInter
+
+variable (s : Set E)
+
+theorem subset_absConvexHull : s ⊆ absConvexHull 𝕜 s :=
+  (absConvexHull 𝕜).le_closure s
```

Comment:
I would strongly suggest making `𝕜` and `s` implicit here

Reply:
Done, thanks.